course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,227 @@
+# RQM, ecologized
+
+<!-- vim-markdown-toc GFM -->
+* [On the necessary changes to the RQM syllabus](#on-the-necessary-changes-to-the-rqm-syllabus)
+* [Draft of proposed schedule](#draft-of-proposed-schedule)
+    * [Week 1](#week-1)
+    * [Week 2](#week-2)
+    * [Week 3](#week-3)
+    * [Week 4](#week-4)
+    * [Week 5](#week-5)
+    * [Week 6](#week-6)
+    * [Week 7](#week-7)
+    * [Week 8-12](#week-8-12)
+* [Labs and lectures from Martin](#labs-and-lectures-from-martin)
+    * [Structured population dynamics](#structured-population-dynamics)
+    * [Predator prey population dynamics](#predator-prey-population-dynamics)
+    * [Lectures](#lectures)
+* [The original RQM course](#the-original-rqm-course)
+    * [Lecture topics](#lecture-topics)
+    * [Discussions](#discussions)
+    * [Exercises](#exercises)
+
+<!-- vim-markdown-toc -->
+
+## On the necessary changes to the RQM syllabus
+- To tailor the course for ecology students, we need to remove some of the existing material.
+- I have tried to summarize the differences between the original schedule and the schedule I propose further down. I did this by classifying all the lectures and tutorials into the overarching themes of the RQM course, and quantified the number of classes dedicated to a theme in the original and proposed schedules (original -> proposed). This can serve as a rough overview for how the course direction is changing:
+    - 13 -> 13 Reproducibility (includes all R classes)
+    - 6 -> 1 Openness in data and science
+    - 5 -> 1 Authorship and publishing
+    - 5 -> 8 Project work
+    - 0 -> 6 Ecology
+- Most of the topics in Christie's course involve both practical and theoretical aspects. For example, all of the R-coding would fall under reproducibility, but there are also discussions dedicated to reproducibility.
+- Note that there are 2 or 3 of these topics per week in the original RQM schedule. Time-wise, this means that a topic approximately equals one class (lecture or tutorial). There are 29 topics here and will be 24 classes taught, so we will need to include three per week in some cases or reduce this to from 29 to 24.
+
+## Draft of proposed schedule
+- An immediate conflict to solve is how to ensure that the students get started with their project early, but also have the necessary knowledge about data analyses, R, and the ecology models that Martin wants to incorporate.
+- As we are now incorporating some ecology classes into this schedule, I think it is important to make sure enough time is dedicated to teaching sound principles around the data analyse pipeline, as this is still the main focus of the course imo.
+- Another point of consideration is whether we should teach all the R stuff with a common data set, so that we can streamline it for teaching the principles, and then hand out student data sets after a few weeks.
```

Comment:
Yes, we should definitely teach the R basics with a common dataset, Software Carpentry style. I don't think we could do it any other way.